### DISCUSSION TITLE
What do I use for an admin key?

### DISCUSSION THREAD

**Discussion by chairmanfaust** [meshtastic/Discussions#103] [General]
I have several devices including a station G2 that will serve as my base station. I have several t-beam devices that i would like to remote admin. I was familiar enough with the legacy admin channel but have moved to v2.5 and would like to try the new remote admin method. I see both my G2 and T-beam have the same public key and the blank admin key field says "The public key authorized to send admin messages to this node" is this a typo? should i enter the private key from my base station on all nodes i want to administer?

**Comment by garthvh** [comment]
The public key from the node that you want to use to administer the node.

**Reply by chairmanfaust** [reply]
i figured it out, page was not refreshing between the two nodes... makes total sense now, time for bed... long day
